### ISSUE TITLE
Autocomplete with Tab and without Popup

### ISSUE THREAD

**Issue by bytecod3r** [controlsfx/controlsfx#1215]
Hi there,

Is there anyway to make the TextField autocomplete without popup? My end goal is to make such thing, that has no pop up and the suggested text shows as light grey, also suggestions can be switched by pressing Tab. 

Please let me know if controlsfx can do it or you are aware of any other library or method that can help.

Many thanks.

![image](https://user-images.githubusercontent.com/8350933/69268038-5942a400-0c09-11ea-8e7c-08b04e0deccb.png)

**Comment by Maxoudela** [comment]
I don't think we have that in ControlsFX and I'm not aware of a Control that does it.

Event with a normal TextField, you can add a promptText, but as soon as you click in the TextField, the promptText is hidden. Whereas in most web textField, the promptText is hidden when you begin to type something.
